### PR TITLE
visionOS Support

### PR DESCRIPTION
**Pull request by loganblevins**
https://github.com/growthbook/growthbook-swift/issues/51

### CODE REVIEW

**Comment by loganblevins** [comment]
@vazarkevych Please approve CI checks so I can assert this works

**Comment by loganblevins** [comment]
Currently fixing an issue where the vision OS framework is looking like it's for iOS .. odd.... so it conflicts where the XCFramework is being exported.

**Comment by loganblevins** [comment]
[GrowthBook.xcframework.zip](https://github.com/growthbook/growthbook-swift/files/14609727/GrowthBook.xcframework.zip)

Example XCFramework output from my build script.

**Comment by loganblevins** [comment]
Looks like I need y bump Xcode tool chain to 15.3...will try this shortly

**Comment by loganblevins** [comment]
@vazarkevych Please approve runner

**Comment by loganblevins** [comment]
I fixed tests please approve runner @vazarkevych

File: Scripts/runTest.sh [hunk 1]
```diff
@@ -1,11 +1,5 @@
 #!/bin/bash
 
-xcodebuild -project GrowthBook-IOS.xcodeproj \
-   -scheme GrowthBook \
-   -sdk iphonesimulator \
-   -destination 'platform=iOS Simulator,name=iPhone 13,OS=16.2'
```
Comment (loganblevins) [inline Scripts/runTest.sh:6]: This old phone ndeinstation wasn't available on Xcode 15.2 .. that was the issue. so instead the CLI tried to use macOS as destination which was a problem

**Comment by loganblevins** [comment]
@vazarkevych  checks pass. please merge

**Review by vazarkevych — APPROVED** [review]

**Pull request merged**